### ISSUE TITLE
Dump normalized longitudinal coordinates

### ISSUE THREAD

**Issue by kyrsjo** [SixTrack/SixTrack#621]
When DUMP is normalizing longitudinal coordinates, it uses (E-E0)/E0 for the raw energy coordinate. This may not be correct - to be checked what is actually used in the matrices.

**Comment by kyrsjo** [comment]
Also of interest to @jfwagner who is using TAS to normalize input coordinates.

**Comment by tpersson** [comment]
Yeah, it does indeed not look to be correct. Also the time like variable seems to be wrong. This will only have an impact for low energy machines. I will try to fix it.
